src/app/modules/group/components/pending-request/pending-request.component.ts

```diff
@@ -103,7 +104,7 @@ export class PendingRequestComponent implements OnInit, OnChanges {
   private parseResults(data: Map<string, any>[]): Result {
     const res : Result = {countRequests: 0, countSuccess: 0};
     data.forEach(elm => {
-      res.countRequests += elm.size,
+      res.countRequests += elm.size;
```

Comment:
A little mistake I come across, I'm surprised it worked before

Reply:
The same way you can write `const a=1, b=2;`, this was a valid syntax as well.

Reply:
Ok thanks